src/index.js

```diff
@@ -0,0 +1,34 @@
+import './styles.css';
+import addList from './addToList.js';
+
+const todoList = document.querySelector('#todoList');
+const inputFaild = document.querySelector('#inputFaild');
+// const clearAll = document.querySelector('#clearAll');
```

Comment:
- [ ] [OPTIONAL] The previous CR also made a suggestion regarding commented out code, because the CR didn't mention this specific part of the code this is an optional suggestion, however, I do strongly suggest addressing this and checking the entire project when something like this is suggested as it will keep your codebase a lot cleaner and more readable.

Reply:
you will find all its requirements in this section sir:
<div class="list-todo" id="${todo.index}"> <input class="form-check-input" type="checkbox" value="" ${todo.completed ? 'checked' : ''} id="defaultCheck1"/> <div class="description ${todo.completed ? 'checked' : ''}" contenteditable="${!todo.completed}">${todo.description}</div> <i class="fas fa-trash delete"></i> </div> ;
From where each element in my array has its project index.